.github/workflows/pull-request.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         uses: actions/setup-go@v2
         with:
           # NOTE: Keep the version in sync with Go toolchain in WORKSPACE.
-          go-version: '1.19.3'
+          go-version: '1.21.1'
```

Comment:
go-toolset in the red hat catalog is currently based on 1.19
jumping to 1.21 might be too fast, how about jumping to 1.20?